sklearn/utils/fixes.py

```diff
@@ -103,7 +103,7 @@ def divide(x1, x2, out=None, dtype=None):
 except TypeError:
     # Compat where astype accepted no copy argument
```

Comment:
I did the check on numpy 1.6.1 (which is the version of numpy shipped by ubuntu 12.04 and that is the oldest version of numpy supported by sklearn) and I got:

```
>>> a.astype(np.float32, copy=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: astype() takes no keyword arguments
```

we have a travis config for that. So this case is properly tested.